app/models/pipeline_run.rb

```diff
@@ -285,6 +300,41 @@ def db_load_ercc_counts
     update(total_ercc_reads: ercc_counts_array.map { |entry| entry[:count] }.sum)
   end
 
+  def db_load_amr_counts
+    amr_results = PipelineRun.download_file(s3_file_for("amr_counts"), local_amr_full_results_path)
+    unless File.zero?(amr_results)
+      amr_counts_array = []
+      # First line of output file has header titles, e.g. "Sample/Gene/Allele..." that are extraneous
+      # that we drop
+      File.readlines(amr_results).drop(1).each do |amr_result|
+        amr_result_fields = amr_result.split(",").drop(2)
+        amr_counts_array << { gene: amr_result_fields[0],
+                              allele: amr_result_fields[1],
+                              coverage: amr_result_fields[2],
+                              depth:  amr_result_fields[3],
+                              drug_family: amr_result_fields[12] }
+      end
+      update(amr_counts_attributes: amr_counts_array)
+    end
+  end
+
+  # Can remove as aggregation is being done on the UI side,
+  # but leaving as useful function for possible future changes (e.g. if UI table)
+  def summarize_amr_drug_family_counts
```

Comment:
Since it is not being used, please remove it

Reply:
done